platform/solr/solr-dependencies/pom.xml

```diff
@@ -23,6 +23,13 @@
     <name>DDF :: Platform :: Solr :: Dependencies</name>
     <description>Shaded jar of Solr core dependencies.</description>
     <packaging>jar</packaging>
+    <properties>
+        <io.dropwizard.metrics.version>3.1.2</io.dropwizard.metrics.version>
+        <info.ganglia.gmetric4j.version>1.0.7</info.ganglia.gmetric4j.version>
+        <org.slf4j.version>1.7.7</org.slf4j.version>
```

Comment:
I attempted to upgrade slf4j, but ran into some issues.  This is one of them (see comment by @millerw8): https://jira.qos.ch/browse/CAL-9.  I think it would be better to work this under a different ticket since the fix is not as trivial as changing the version at https://github.com/codice/ddf/blob/master/pom.xml#L82.  Thoughts?